pkg/controller/tidbcluster/pod_control.go

```diff
@@ -340,3 +340,14 @@ func (c *PodController) syncTiKVPod(ctx context.Context, pod *corev1.Pod, tc *v1
 
 	return reconcile.Result{}, nil
 }
+
+func needEvictLeader(pod *corev1.Pod) (string, string, bool) {
+	for _, key := range v1alpha1.EvictLeaderAnnKeys {
+		value, exist := pod.Annotations[key]
+		if exist {
+			return key, value, true
```

Comment:
any restriction to multi evict keys (with differet values)?

Reply:
no, just use a default priority.